src/helpers/abis/erc1155.json

```diff
@@ -0,0 +1,314 @@
+[
```

Comment:
We should add only ABI for the functions that we use, also we should use the string ABI it's much more compact, see this code to turn ABI to this compact mode https://github.com/snapshot-labs/snapshot.js/blob/master/test/abi.ts